### DISCUSSION TITLE
DJ Systemデザインロードマップ

### DISCUSSION THREAD

**Discussion by txxxxc** [lit-kansai/dj_system-docs#43] [ドキュメント]
## デザインでつくるもの

- [ ] 1.トンマナ
   - 一次〆切: 10月末
- [ ] 2.カラーテーマ（ベース1色 + 8色）
   - 一次〆切: 10月末
- [ ] 3.ロゴ（ファビコンと一緒）
   - 一次〆切: 11月末
- [ ] 4.メンバーのフロントデザイン（④スマホ→⑤PC）
   - 検索画面
   - 検索結果
   - リクエストするときのモーダル
   - リクエスト完了時のモーダル
   - 時間制限待ってるとき
   - 〆切: 年内
- [ ] 5.OGP画像
   - 〆切: 1月末
- [ ] 6.404のページ
   - 〆切: 1月末

**Comment by gassi4731** [comment]
議論したいこと
- トンマナについて方向性
- ロゴの方向性

**Comment by txxxxc** [comment]
2022/11/28更新
## デザインでつくるもの

- [ ] 1.トンマナ
   - 一次〆切: 12月末
- [ ] 2.カラーテーマ（ベース1色 + 8色）
   - 一次〆切: 12月末
- [ ] 3.ロゴ（ファビコンと一緒）
   - 一次〆切: 12月末
- [ ] 4.メンバーのフロントデザイン（④スマホ→⑤PC）
   - 検索画面
   - 検索結果
   - リクエストするときのモーダル
   - リクエスト完了時のモーダル
   - 時間制限待ってるとき
   - 〆切: 保留
- [ ] 5.OGP画像
   - 〆切: 保留
- [ ] 6.404のページ
   - 〆切: 保留
